docker-compose.yml

```diff
@@ -64,6 +64,12 @@ services:
         - SKIP_TESTS=true
         - SPLUNK_URL=${SPLUNK_URL:-http://splunk:8088}
         - SPLUNK_TOKEN=${SPLUNK_HEC_TOKEN:-token}
+        - REDIS_HOST=redis
+        - REDIS_PORT=6379
+        - REDIS_PASSWORD=password
```

Comment:
Should we get these from the env variables as well? For example; 
REDIS_PASSWORD=${REDIS_PASSWORD:-password}

Reply:
I would like to understand the use case of over ridding this in docker-compose.yaml.  The direction I was going is just use sensible defaults for local development only.

Reply:
I was just thinking about a scenario if the password or host value would be different for dev or test environment.

Reply:
We dont use docker-compose when we deploy to OpenShift.

Reply:
Oh, I see. Makes sense. You may just ignore my comment then.